### PR TITLE
Update absl-py to version 0.15.0

### PR DESCRIPTION
**Pull request by varlackc**
absl-py 0.15.0
1. check the upstream
    https://github.com/abseil/abseil-py/tree/pypi-v0.15.0
    
2. check the pinnings
    https://github.com/abseil/abseil-py/blob/pypi-v0.15.0/setup.py
    

3. check the changelogs
    There were no change logs so the code base was explored using the release information
    https://github.com/abseil/abseil-py/compare/pypi-v0.15.0...v1.0.0

4. additional research
    http://github.com/conda-forge/absl-py-feedstock/issues

    No issues at the time of the review

5. verify dev_url
    https://github.com/abseil/abseil-py

6. verify doc_url
    https://abseil.io/docs/

7. pip in the test section
8. veriy the test section
9. additional tests

In order to test the `absl-py` package version `0.15.0` the folowing
command was used:
`conda build absl-py-feedstock --test`
the test result was the following:
`All tests passed`

Based on the research findings and the test results we can conclude
that it is safe to update `absl-py` to version `0.15.0`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {% set name = "absl-py" %}
-{% set version = "0.13.0" %}
+{% set version = "0.15.0" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:2]: The latest version is 1.0.0

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,6 +15,8 @@ source:
 
 build:
   number: 0
+  # the package absl is not available in s390x
+  skip: True  # [py<36 or s390x]
   noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: In host use `setuptools >=24.2`, see https://github.com/abseil/abseil-py/blob/0f86230d77cd90ea1236f0cbee4df9a9b290ef66/setup.py#L42

Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: In run use `python >=3.6`

Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: In the test/requires should be `python <3.10`

Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: The license name should be: `Apache-2.0`

**Review by skupr-anaconda — CHANGES_REQUESTED** [review]

**Comment by skupr-anaconda** [comment]
@varlackc Should we merge or close this PR?

**Comment by varlackc** [comment]
> @varlackc Should we merge or close this PR?

The PR has been merged

**Pull request merged**